src/de/roderick/weberknecht/WebSocketMessage.java

```diff
@@ -18,12 +18,16 @@
 
 public class WebSocketMessage {
   private Byte[] message;

```

Comment:
This definitely predates you PR - so no changes necessary - but I wonder why we use `Byte` here instead of `byte`.